test/test_utils.py

```diff
@@ -722,7 +731,96 @@ def test_load_commit_from_forked_repo(self):
         with self.assertRaisesRegex(
                 ValueError,
                 'If it\'s a commit from a forked repo'):
-            model = torch.hub.load('pytorch/vision:4e2c216', 'resnet18', force_reload=True)
+            model = torch.hub.load('pytorch/vision:4e2c216', 'resnet18', force_reload=True, trust_repo=True)
+
+    def _cleanup(self):
+        # cleanup
+        folder_path = os.path.join(torch.hub.get_dir(), 'ailzhang_torchhub_example_master')
+        if os.path.exists(folder_path):
+            shutil.rmtree(folder_path)
+
+        file_path = os.path.join(torch.hub.get_dir(), 'trusted_list')
+        if os.path.exists(file_path):
+            with open(file_path, "r") as f:
+                lines = f.readlines()
+            with open(file_path, "w") as f:
+                for line in lines:
+                    if line.strip("\n") != "ailzhang_torchhub_example":
+                        f.write(line)
+
+        file_path_legacy = os.path.join(torch.hub.get_dir(), 'trusted_list_legacy')
+        if os.path.exists(file_path_legacy):
+            with open(file_path_legacy, "r") as f:
+                lines = f.readlines()
+            with open(file_path_legacy, "w") as f:
+                for line in lines:
+                    if line.strip("\n") != "ailzhang_torchhub_example_master":
+                        f.write(line)
+
+    @retry(Exception, tries=3)
+    @patch('torch.hub._get_trusted_input', return_value='')
+    def test_untrusted_repo(self, input):
```

Comment:
Here and in other tests it looks like the `input` parameter isn't used

```suggestion
    def test_untrusted_repo(self):
```